### ISSUE TITLE
add a exclude function for folders/files

### ISSUE THREAD

**Issue by Daniel-from-Germany** [mergebase/log4j-detector#11]
add a exclude function for folders/files

**Comment by chgad** [comment]
This maybe especially goof for those issues here #25 and #30

**Comment by beckerr-rzht** [comment]
The problem could be solved with `find` and the option `--stdin` which I added.
See #42

Example:
```
find / -xdev -type f -not -path "/skip/this/path/*" | java -jar log4j-detector-2021.12.17.jar --stdin
```

**Comment by juliusmusseau** [comment]
implemented in v2021.12.20
